addons/website_forum/views/website_forum.xml

```diff
@@ -611,7 +611,7 @@
                             <t t-if="no_filters"> All</t>
                             <t t-elif="filters == 'solved'"> Solved</t>
                             <t t-elif="filters == 'unsolved'"> Unsolved</t>
-                            <t t-elif="filters == 'unanswered'"> Unsolved</t>
+                            <t t-elif="filters == 'unanswered'"> Unanswered</t>
```

Comment:
Bug fix ?

Reply:
Yes, I saw it when I was hiding "Solved" and "Unsolved" filter for a "Discussion" forum. What should I do in this case ?

Reply:
Again: bug fix in the oldest branch which has the bug.